decidim-generators/lib/decidim/generators/app_templates/initializer.rb

```diff
@@ -1,488 +0,0 @@
-  #     @mobile_phone_number = mobile_phone_number
-  #     @code = code
-  #   end
-  #
-  #   def deliver_code
-  #     # Actual code to deliver the code
-  #     true
-  #   end
-  # end
-  #
-  # config.sms_gateway_service = "MySMSGatewayService"
-
-  # Timestamp service configuration
-  #
-  # Provide a class to generate a timestamp for a document. The instances of
-  # this class are initialized with a hash containing the :document key with
-  # the document to be timestamped as value. The instances respond to a
-  # timestamp public method with the timestamp
-  #
-  # An example class would be something like:
-  #
-  # class MyTimestampService
-  #   attr_accessor :document
-  #
-  #   def initialize(args = {})
-  #     @document = args.fetch(:document)
-  #   end
-  #
-  #   def timestamp
-  #     # Code to generate timestamp
-  #     "My timestamp"
-  #   end
-  # end
-  #
-  #
-  # config.timestamp_service = "MyTimestampService"
-
-  # PDF signature service configuration
-  #
-  # Provide a class to process a pdf and return the document including a
-  # digital signature. The instances of this class are initialized with a hash
-  # containing the :pdf key with the pdf file content as value. The instances
-  # respond to a signed_pdf method containing the pdf with the signature
-  #
-  # An example class would be something like:
-  #
-  # class MyPDFSignatureService
-  #   attr_accessor :pdf
-  #
-  #   def initialize(args = {})
-  #     @pdf = args.fetch(:pdf)
-  #   end
-  #
-  #   def signed_pdf
-  #     # Code to return the pdf signed
-  #   end
-  # end
-  #
-  # config.pdf_signature_service = "MyPDFSignatureService"
-
-  # Etherpad configuration
-  #
-  # Only needed if you want to have Etherpad integration with Decidim. See
-  # Decidim docs at https://docs.decidim.org/en/services/etherpad/ in order to set it up.
-  #
-  if Decidim::Env.new("ETHERPAD_SERVER").present? && Decidim::Env.new("ETHERPAD_API_KEY").present?
-    config.etherpad = {
-      server: Decidim::Env.new("ETHERPAD_SERVER").to_s,
-      api_key: Decidim::Env.new("ETHERPAD_API_KEY").to_s,
-      api_version: Decidim::Env.new("ETHERPAD_API_VERSION", "1.2.1").to_s
-    }
-  end
-
-  # Sets Decidim::Exporters::CSV's default column separator
-  # config.default_csv_col_sep = Decidim::Env.new("DECIDIM_DEFAULT_CSV_COL_SEP", ";").to_s
-
-  # The list of roles a user can have, not considering the space-specific roles.
-  # config.user_roles = %w(admin user_manager)
-
-  # The list of visibility options for amendments. An Array of Strings that
-  # serve both as locale keys and values to construct the input collection in
-  # Decidim::Amendment::VisibilityStepSetting::options.
-  #
-  # This collection is used in Decidim::Admin::SettingsHelper to generate a
-  # radio buttons collection input field form for a Decidim::Component
-  # step setting :amendments_visibility.
-  # config.amendments_visibility_options = %w(all participants)
-
-  # Machine Translation Configuration
```

Comment:
For consistency with the other external services, this should be moved to a page like https://docs.decidim.org/en/v0.30/services/sms (i.e. https://docs.decidim.org/en/v0.30/services/machine_translations)